src/Editor/GUI/Editors/EntityEditor.cpp

```diff
@@ -76,6 +77,44 @@ void EntityEditor::Show() {
         ImGui::Text("Transform");
         ImGui::ShowHelpMarker("The entity's position, rotation and scale.", 75.f);
         ImGui::Indent();
+
+        if (Hymn().gridSettings.gridSnap) {
+
```

Comment:
Not sure about this line break; style doesn't say anything explicitly so I don't know how we want it.